rllib/evaluation/tests/test_rollout_worker.py

```diff
@@ -484,20 +485,31 @@ def step(self, action):
 
     def test_reward_clipping(self):
         # Clipping: True (clip between -1.0 and 1.0).
+        config = (
+            AlgorithmConfig()
+                .rollouts(num_rollout_workers=0, batch_mode="complete_episodes")
+                .environment(clip_rewards=True)
+        )
         ev = RolloutWorker(
             env_creator=lambda _: MockEnv2(episode_length=10),
             default_policy_class=MockPolicy,
-            config=AlgorithmConfig()
-            .rollouts(num_rollout_workers=0, batch_mode="complete_episodes")
-            .environment(clip_rewards=True),
+            config=config,
         )
+        sample = convert_ma_batch_to_sample_batch(ev.sample())
         ws = WorkerSet._from_existing(
             local_worker=ev,
             remote_workers=[],
         )
-        self.assertEqual(max(ev.sample()["rewards"]), 1)
+        self.assertEqual(max(sample["rewards"]), 1)
         result = collect_metrics(ws, [])
-        self.assertEqual(result["episode_reward_mean"], 1000)
+        # Shows different behavior when connector is on/off.
+        if config.enable_connectors:
+            # episode_reward_mean shows the correct clipped value.
```

Comment:
Uh, nice!

Reply:
somewhere random man.
as soon as I see this, I was like "let's get this finished and turned on" :)